cirq/work/sampler.py

```diff
@@ -199,3 +199,57 @@ async def run_sweep_async(
             An awaitable TrialResult.
         """
         return self.run_sweep(program, params=params, repetitions=repetitions)
+
+    def run_batch(
+            self,
+            programs: List['cirq.Circuit'],
+            params_list: Optional[List['cirq.Sweepable']] = None,
+            repetitions: Union[int, List[int]] = 1,
+    ) -> List['cirq.TrialResult']:
```

Comment:
Same comment here.  I think flattening the list is probably a bad idea for the main interface.

Reply:
I agree; I'll change it to a list of lists.

Reply:
Done.